syntax = "proto3";

service CacheService {
  rpc GetValue (CacheRequest) returns (CacheResponse) {}
  rpc SetValue (CacheData) returns (CacheResponse) {}
}

message CacheRequest {
  string key = 1;
}

message CacheData {
  string key = 1;
  string value = 2;
}

message CacheResponse {
  string value = 1;
  string status = 2;
}
